challenges/binary-search/test_binary_search.py

```diff
@@ -0,0 +1,9 @@
+import binary_search as bs
+
+
+def test_BinarySearch():
+    assert bs.BinarySearch([1, 2, 3, 4, 5], 5) == 4
+
+    assert bs.BinarySearch([1, 2, 3, 4, 5], 8) == -1
+
+    assert bs.BinarySearch([], 1) == -1
```

Comment:
Make each of these their own test function. You are testing three different things, so each should be its own test.